style.css

```diff
@@ -419,14 +420,176 @@ input:focus {
   border-color: rgb(98, 108, 201);
 }
 
+.headerlish {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
```

Comment:
Why do you have two media queries in your code apart from desktop version. You can kindly look into this
![ClipWindowsGIF](https://user-images.githubusercontent.com/58138876/229720841-376c9e07-8347-4649-b2f8-a6dc292c0a5c.gif)